src/main/scala/rps/Game.scala

```diff
@@ -0,0 +1,63 @@
+package rps
+
+import scala.io.StdIn.readLine
+import scala.util.Random
+
+object Game {
+
+  // TODO this should be an enum (or Scala equivalent of TS Enum)
+  object Weapon {
+    val Rock: String = "0" // 🪨
+    val Paper: String = "1" // 📄
+    val Scissors: String = "2" // ✂️
+  }
+
+  // TODO this should be an enum (or Scala equivalent of TS Enum)
+  object Winner {
+    val Draw: String = "0" // It's a draw
+    val User: String = "1" // User wins
+    val Cpu: String = "2" // Cpu wins
+  }
+
+  def play(): Unit = {
+    println(s"""Choose your weapon:
+    ${Weapon.Rock} - 🪨 Rock
+    ${Weapon.Paper} - 📄 Paper
+    ${Weapon.Scissor} - ✂️ Scissor
```

Comment:
Come dicevamo venerdì manca da aggiustare Scissor -> Scissors in giro

Reply:
Adesso pusho il change